src/synthesizer/particle/stars.py

```diff
@@ -1605,6 +1605,80 @@ def get_particle_spectra_reprocessed(
 
         return reprocessed
 
+    def get_particle_spectra_screen(
```

Comment:
Could we not just use a generic attenuated method and include a `label` argument? Otherwise, we will have to derive a bunch of different spectra methods like the integrated version which encompasses all possible masks etc. 

This is why its been left on `Sed`s up until now, combining different masks and spectra is really non-trivial without introducing lots of code.

Reply:
Yes, added label.

Reply:
Cool. One remaining concern is how best to provide an interface to combine different attenuated spectra (i.e. old and young) generated via this.